### ISSUE TITLE
Would tangential topics (e.g. re: research and write-ups of projects) make sense here, as practices?

### ISSUE THREAD

**Issue by RamblingCookieMonster** [PoshCode/PowerShellPracticeAndStyle#70]
Hi!

TLDR: Would it make sense to include topics tangential to code, but important to code writers here?

Examples (whether you agree or not):
- Conduct a mini "literature review" before undertaking and releasing a project.  Are there similar projects out there?  Is there a reason to create your own, rather than contributing to these?  Did you borrow ideas or code from these?  Consider listing these at a minimum in any blog posts, readmes, and help content, and ideally, explain scenarios where your solution is a better fit.
- Reduce dependence on the current state and time.  Include a synopsis of your operating environment such as PowerShell version, operating system (if applicable), module versions, etc.  When linking to content that may change, attempt to use a static link: for example, when linking to content in GitHub or a similar site, [specify a specific commit](https://twitter.com/psCookieMonster/status/757372331362779136), rather than the generic "latest" link.

etc.

Granted, these aren't PowerShell specific, so maybe this isn't the best spot?  Thoughts?

Cheers!

**Comment by Jaykul** [comment]
If anyone wanted to write some stuff like that, I'd be happy to add a section on general development best practices 😉 but I don't think I'm going to write it, at this point
